kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/ProcessServiceIntegrationTest.java

```diff
@@ -353,6 +403,32 @@ public void testManipulateProcessVariable() throws Exception {
         }
 
     }
+    
+    public void testManipulateProcessVariableWrongContainer() throws Exception {
+        Long processInstanceId = processClient.startProcess(CONTAINER_ID, PROCESS_ID_SIGNAL_PROCESS);
+
+        assertNotNull(processInstanceId);
+        assertTrue(processInstanceId.longValue() > 0);
+
+        try {
+            Object personVar = null;
+            try {
+                personVar = processClient.getProcessInstanceVariable(CONTAINER_ID, processInstanceId, "personData");
+                fail("Should fail as there is no process variable personData set yet");
+            } catch (KieServicesException e) {
+                // expected
+            }
+            assertNull(personVar);
+
+            personVar = createPersonInstance(USER_JOHN);
+            exceptionRule.expect(KieServicesException.class);
```

Comment:
I would add below to make sure `personVar` is not `null`
```suggestion
            assertNotNull(personVar);
            exceptionRule.expect(KieServicesException.class);
```